airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/StandardSourceTest.java

```diff
@@ -73,9 +73,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class TestSource {
+public abstract class StandardSourceTest {
```

Comment:
renaming to remove ambiguity

Reply:
ignore this is it's too annoying, but I think the slightly more semantically clear name is SouceStandardTest. It is the "standard tests" that we run for "sources", not the a "standard" version of "source test" (the point being that "source test" isn't really a primitive in our vocab, while "standard tests" and "source" both are).